auth/src/auth.cc

```diff
@@ -373,5 +373,13 @@ AUTH_RESULT_FN(Auth, SignInWithEmailAndPassword, AuthResult)
 
 AUTH_RESULT_FN(Auth, CreateUserWithEmailAndPassword, AuthResult)
 
+#if !FIREBASE_PLATFORM_IOS
+// Stub for non-iOS platforms.
+void Auth::UseUserAccessGroup(const char* user_access_group) {
```

Comment:
This function isn't necessary in this file, since it's covered in all 3 platform-specific files.